### PR TITLE
Issue #67 - Looking for corruption on the Federal Budget

### PR DESCRIPTION
**Pull request by marcusrehm**
Related to issue #67 and maybe #167 .

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 __pycache__/
 config.ini
 data
+.idea/
+serenata-de-amor.iml
```
Comment (cuducos) [inline .gitignore]: Just out of curiosity — what is this file about?

Comment (marcusrehm) [inline .gitignore]: Sorry @cuducos ! At beginning I was using IntelliJ IDE and this file is generated automatically with some IDE project's properties.

Do you want me to remove that?

Comment (cuducos) [inline .gitignore]: No way. more people can use IntelluJ IDE too ; )
It was just my curiosity…

Comment (marcusrehm) [inline .gitignore]: Ok! :)

**Pull request merged**